alvr/client_openxr/src/stream.rs

```diff
@@ -548,7 +549,19 @@ fn stream_input_loop(
                 &int_ctx.face_sources,
                 stage_reference_space,
                 now,
-            ),
+            )
+            .map(|option| {
+                option.map(|pose| {
+                    if int_ctx.platform.is_pico() {
+                        let mut relative_head_pose = int_ctx.relative_head_pose;
+                        relative_head_pose.position = Vec3::ZERO;
+
+                        relative_head_pose * pose
```

Comment:
I think you don't need to set the position to zero

Reply:
I see... well then you could undo the last commit, but keep the comment in the struct, and specify this detail